pkg/naming/labels.go

```diff
@@ -91,9 +91,10 @@ func mergeLabels(l1, l2 map[string]string) map[string]string {
 	return res
 }
 
-func ManagedResourcesLabels(managingClusterDomain string) map[string]string {
+func RemoteManagedResourcesLabels(managingClusterDomain string) map[string]string {
 	return map[string]string{
-		ManagedByClusterLabel: managingClusterDomain,
+		"app.kubernetes.io/managed-by": RemoteOperatorAppNameWithDomain,
```

Comment:
Can we add `"app.kubernetes.io/managed-by"` to `pkg/naming/constants.go`?

Reply:
done